rsts/concepts/projects.rst

```diff
@@ -2,8 +2,10 @@
 
 Projects
 ========
-A project in Flyte is a grouping of workflows and tasks to achieve a particular goal. A Flyte project can map to an engineering project or everything that's owned by a team or an individual. There cannot be multiple projects with the same name in Flyte. 
+A project in Flyte is a grouping of :ref:`workflows <divedeep-workflows>` and :ref:`tasks <divedeep-tasks>` to achieve a particular goal. 
 
-Since the fully-qualified name for tasks and workflows include project name and domain name, the task/workflow names are only required to be unique within a project. The workflows in a project A can refer tasks and workflows from other projects using the fully-qualified name.
+A Flyte project can map to an engineering project or everything that's owned by a team or an individual. There cannot be multiple projects with the same name in Flyte. 
 
-Flyte allows users to set resource limit for each project. Flyte provides basic reports and dashboards automatically for each project. The information captured in these reports include workflow/task level insights, resource usage and billing information.
\ No newline at end of file
+Since the fully-qualified name for tasks and workflows include the project name and domain name, the task/workflow names are only required to be unique within a project. The workflows in a project ``A`` can refer tasks and workflows from other projects using the fully-qualified name.
```

Comment:
```suggestion
Since the fully-qualified name for tasks and workflows include the project and domain name, the task/workflow names are only required to be unique within a project. The workflows in a project ``A`` can refer to tasks and workflows in other projects using the fully-qualified name.
```